docs/NATS.md

```diff
@@ -0,0 +1 @@
+# NATS
```

Comment:
Let's just populate this with a very basic (autogenerated) summary of "What is NATS", and then a section on connecting to our NATS where we just say please contact us for help with connecting/authenticating to our nats service